{{ cookiecutter.package_name }}/.pre-commit-config.yaml

```diff
@@ -20,3 +20,6 @@ repos:
         exclude: ".*(.fits|.fts|.fit|.header|.txt|tca.*|.json)$|^CITATION.rst$"
       - id: mixed-line-ending
         exclude: ".*(.fits|.fts|.fit|.header|.txt|tca.*)$"
+ci:
+  autofix_prs: false
+  autoupdate_schedule: "quarterly"
```

Comment:
Won't this trigger on the other repos tho?

Reply:
It doesn't seem possible to disable it. https://pre-commit.ci/#configuration

But anyway we don't have autoupdates working here yet.